builder/vmware/common/step_create_disks.go

```diff
@@ -45,8 +45,13 @@ func (s *StepCreateDisks) Run(ctx context.Context, state multistep.StateBag) mul
 	}
 	// Additional disks
 	if len(s.AdditionalDiskSize) > 0 {
+		incrementer := 1
 		for i, diskSize := range s.AdditionalDiskSize {
-			path := filepath.Join(*s.OutputDir, fmt.Sprintf("%s-%d.vmdk", s.DiskName, i+1))
+			// scsi slot 7 is reserved, so we skip it.
+			if i+incrementer == 7 {
+				incrementer = 2
+			}
+			path := filepath.Join(*s.OutputDir, fmt.Sprintf("%s-%d.vmdk", s.DiskName, i+incrementer))
```

Comment:
This is going to increment by two after 7 so: 5,6,8,10,12. Not sure if that's what you expected, here's a suggestion:

```go
diskNmbr := 1
for _, diskSize := range s.AdditionalDiskSize {
	// scsi slot 7 is reserved, so we skip it.
	if diskNmbr == 7 {
		diskNmbr
	}
	path := filepath.Join(*s.OutputDir, fmt.Sprintf("%s-%d.vmdk", s.DiskName, diskNmbr))
	// do stuff ...
	i++
}
```

Reply:
https://play.golang.org/p/8lBuhiBzbeE

Reply:
oh right ! Yup too fast ! sorry :D